packages/js/github-actions/actions/coverage-report/action.yml

```diff
@@ -0,0 +1,69 @@
+name: Coverage Report
```

Comment:
Maybe is worth it to change the defaults to include the PHP keyword as we did in AW implementation. So we can save to add them in our implementations over different plugins. 

         ```
          report-name: php-coverage-report
          report-path: tests/php-coverage
          test-comment: "PHP unit test coverage"
```

Reply:
I originally created this PR when I added it for GLA, which does both PHP and JS unit tests. So I specifically wanted to leave it independent from the language, as it should be usable for either.